### DISCUSSION TITLE
Can you share documentation that how you collect ISO?

### DISCUSSION THREAD

**Discussion by umlumpa** [Security-Onion-Solutions/securityonion#12112] [General]
Can you share documentation that how you collect ISO? I want collect ISO images manually! could you mind share your documentation with me?

**Comment by reyesj2** [comment]
I saw your previous discussion post #12108. Can you explain a little what you're trying to do? That way I can try to point you in the right direction.

If you download the Security Onion ISO you can do a full install of Security Onion without access to the internet using airgap setup. https://docs.securityonion.net/en/2.4/airgap.html?highlight=airgap#airgap
